### ISSUE TITLE
Set up remark to use roughly the same settings/linters as guides-source

### ISSUE THREAD

**Issue by chancancode** [ember-learn/super-rentals-tutorial#13]
That workflow is that we write special "runnable" markdowns in that repo. When we merge a pull request here, CI will build that into "regular" markdown, and integrate the result into the [guides-source](https://github.com/ember-learn/guides-source/) repo. Because we don't have linting setup in super-rentals-tutorial, often we will merge a PR over there, only to find out when it's being integrated that it failed some lints (repeated words and what not), and had to fix that in a second PR on super-rentals-tutorial.

For example:

1. PR here: https://github.com/ember-learn/super-rentals-tutorial/pull/73
2. Pushed to guides-source: https://github.com/ember-learn/guides-source/pull/1126
3. Failed guides-source CI due to "qunit-dom" not being a word in the dictionary
4. Fixed in 7502df14b5267648f8442f590b709c1d9f10e181
5. (guides-source PR updated and CI failure resolved)

It would be nice to "roughly mirror" (i.e. duplicate) the linting setup we used in guides-source into this repo, so the step 1 would have failed CI and we would have fixed it before merging  the PR and avoid step 3, 4, 5.

**Comment by maxwondercorn** [comment]
@chancancode I have the markdown linting mostly complete.  It will be the same linting process that exists in `guides-source` and `cli-quides`.  

Maintaining the local dictionary (`.local.dic`) will be a bit different because of the merge into `guides-source`.  It will still be straightforward and I will document in contributing.md

To complete the work, I need another `guides-soruce` PR to add more dictionary words once PR ember-learn/guides-source#1127 is merged.

One outstanding issue is the undefined links (the TODO's) generating errors.  We can disable that linting rule in the tutorial but it would still fail when merged into `guides-source`. 

The changes required for linting can still be added. You'll just get errors when you run the lint.

**Comment by maxwondercorn** [comment]
I'm creating a PR this afternoon to add the linting to the tutorial.  It's manual linting now but the `lint:md` script command can be added to CI to catch errors before deploying to `guides`.

I added content to CONTRIBUTING.md re: linting and spellchecking specifically.  TLDR: If you add words to the local spelling dictionary `.local.dic` you need to do a PR on `guides-source` to also include them in it's `.local.dic` file.  

This is a bit odd but when the PR is linted in guides-source it uses it's local dictionary.
